src/smt/term_formula_removal.cpp

```diff
@@ -206,14 +240,25 @@ Node RemoveTermFormulas::run(TNode node, std::vector<Node>& output,
   // If changes, we rewrite
   if(somethingChanged) {
     Node cached = nodeManager->mkNode(node.getKind(), newChildren);
-    d_iteCache.insert(cacheKey, cached);
+    d_tfCache.insert(cacheKey, cached);
     return cached;
   } else {
-    d_iteCache.insert(cacheKey, Node::null());
+    d_tfCache.insert(cacheKey, Node::null());
     return node;
   }
 }
 
+Node RemoveTermFormulas::getSkolemForNode(Node node) const
+{
+  context::CDInsertHashMap<Node, Node, NodeHashFunction>::const_iterator itk =
+      d_skolem_cache.find(node);
+  if (itk != d_skolem_cache.end())
+  {
+    return itk->second;
+  }
+  return Node::null();
+}
+
 Node RemoveTermFormulas::replace(TNode node, bool inQuant, bool inTerm) const {
   //if(node.isVar() || node.isConst()){
   //   (options::biasedITERemoval() && !containsTermITE(node))){
```

Comment:
Any reasons why we should not remove this dead code (and the lines below)?

Reply:
I'll remove